projects/ui/src/Components/Apis/ApisTab/ApiSummaryCards/ApiSummaryGridCard.tsx

```diff
@@ -0,0 +1,82 @@
+import { useMemo } from "react";
+import { APIProduct } from "../../../../Apis/api-types";
+import { Icon } from "../../../../Assets/Icons";
+import { DataPairPill, DataPairPillList } from "../../../Common/DataPairPill";
+import { getApiDetailsLink } from "../../helpers";
+import { ApiSummaryGridCardStyles } from "./ApiSummaryGridCard.style";
+
+/**
+ * MAIN COMPONENT
+ **/
+export function ApiSummaryGridCard({ api }: { api: APIProduct }) {
+  // In the future banner images may come through API data.
+  //   Even when that is the case, a default image may be desired
+  //   for when no image is available.
+  // Further, you may have some clever trick for setting one of
+  //   many default images.
+  const defaultCardImage = useMemo(
+    () => {
+      return "https://img.huffingtonpost.com/asset/57f2730f170000f70aac9059.jpeg?ops=scalefit_960_noupscale";
```

Comment:
nit: We might want this to be an image on our server, just so we have control over it (privacy, 404, etc)

Reply:
Agreed. Or at least our website